config/routes.js

```diff
@@ -311,6 +311,16 @@ module.exports.routes = {
   'POST /api/v1/entrances/from-duplicate/:id':
     'v1/entrance-duplicate/create-from-duplicate',
 
+  // Snapshots
+  'GET /api/v1/comments/:id/snapshots': 'v1/comment/get-snapshots',
```

Comment:
@julien-vendran @CecileDestIG : You need to update the Swagger file with all these new routes.
You can use https://editor.swagger.io/ to help you double check your changes on the swagger file.